ci-operator/step-registry/rosa/conf/machine-pool/rosa-conf-machine-pool-commands.sh

```diff
@@ -53,7 +53,7 @@ fi
 
 TAINTS="prowci=true:NoSchedule"
 if [[ ! -z "$MP_TAINTS" ]]; then
-  TAINTS="${TAINTS},${MP_TAINTS}"
+  TAINTS="${MP_TAINTS}"
```

Comment:
@yasun1 I am removing this default taints if `MP_TAINTS` is specified because this `prowci` taint is not tolerated by cluster default prometheus-k8s components and if I try to patch those `statefulsets` I am afraid they get reconciled by HCP operators. is this taint being used/required anywhere ?

Reply:
@yasun1 Waiting for your response for this before merging PR. Changes look good from PerfScale point of view, but not sure if the above change affects other tests. Can you ptal? TIA!